### PR TITLE
Does not work with gnome 3.32

### PR DESCRIPTION
**Pull request by ChrisLauinger77**
converted extension.js with jscodeshift
it still does not work but the error message in log changed to 
TypeError: this._getApps is not a function

### CODE REVIEW

File: messagingmenu@screenfreeze.net/extension.js [hunk 1]
```diff
@@ -258,9 +252,9 @@ const MessageMenu = new Lang.Class({
 			this.menu.addMenuItem(newLauncher);
 		}
 
-	},
+	}
 
-	_getApps: function() {
+    _getApps() {
```
Comment (sinisterstuf) [inline messagingmenu@screenfreeze.net/extension.js]: This changes how `_getApps()` needs to be called. The changes is probably correct, but the caller on line 149 will have to be updated accordingly as well. This is probably only the first of several such errors.

**Comment by ChrisLauinger77** [comment]
I have no idea how _getApps and _buildMenu needs to be called.
I tried to move the call outside of the constructor in function enable like:
`
function enable() {
	_indicator = new MessageMenu;
	_indicator._getApps();
	_indicator._buildMenu();

	originalUpdateCount = MessageTray.SourceActor.prototype._updateCount;
	MessageTray.SourceActor.prototype._updateCount = customUpdateCount;

	statusArea =  Main.panel.statusArea;

	Main.panel.addToStatusArea('messageMenu', _indicator,1);

	iconBox =  statusArea.messageMenu.actor;

	originalStyle = iconBox.get_style();
}

still the same error. all code examples I found tell it should work though :(
`

**Comment by ChrisLauinger77** [comment]
Now the last commit it loads but does not show an icon in upper right corner :(
Close but not quite. 
Any hints where to look for that ?

**Comment by ChrisLauinger77** [comment]
when I use mail-message-new-symbolic as Icon it works - the one from the /icons subdirectory from the extension does not :( 
but i am happy now - extension is loaded and at least an icon is shown

**Comment by ChrisLauinger77** [comment]
> We'll probably have to make a separate branch of the project for this, because although it works with newer versions of GNOME Shell, it won't work with the older versions which worked with the older code.
I would take the current master and create a legacy branch from it. 
After that the new stuff can become master

**Comment by ChrisLauinger77** [comment]
changed metadata.json to only support gnome-shell 3.32 and version to 17

**Comment by sinisterstuf** [comment]
OK, so I just checked and the version you added to metadata.json are already listed in the file in the master branch, so I'll skip the commits that modify that file, they cause conflicts anyway.

As for the rest of it, I tried it out on Arch Linux and it works as expected, I also read up a bit on ES6 classes to make sure the code is alright and it looks good to go, I'll merge this soon and it'll go out in the next release, thanks for your hard work! 💛

**Pull request closed without merging**